### PR TITLE
tools/tpm2_pcrreset: Add command to reset one or more PCR indexes

### LINKED ISSUE #853: Implement pcr_reset
Implement a tool to use the pcr_reset command.

### PR DESCRIPTION
**Pull request by lestums**
Resolve #853

- Added command to makefile
- Added manual entry for tpm2_pcrreset
- Added command relying on ESAPI
- Added integration test for coverage

Signed-off-by: sebastien le stum <lestums@protonmail.com>

### CODE REVIEW

File: test/integration/tests/pcrreset.sh [hunk 1]
```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+#;**********************************************************************;
+#
+# Copyright (c) 2016-2018, Intel Corporation
```
Comment (williamcroberts) [inline test/integration/tests/pcrreset.sh]: I don't have a copyright on this, I didn't author/modify anything. Remove Intel.

File: tools/tpm2_pcrreset.c [hunk 2]
```diff
@@ -0,0 +1,131 @@
+//**********************************************************************;
+// Copyright (c) 2018, Intel Corporation
```
Comment (williamcroberts) [inline tools/tpm2_pcrreset.c]: same

**Comment by williamcroberts** [comment]
Oh please add to your commit message:
```
Fixes: #853
```
https://github.com/tpm2-software/tpm2-tools/issues/853

**Comment by williamcroberts** [comment]
> Oh please add to your commit message:
> 
> ```
> Fixes: #853
> ```
> #853

Nevermind, I see you have it as Resolve. That works too.

**Comment by lestums** [comment]
Done. Also rebased on master

**Comment by williamcroberts** [comment]
Thank you for your excellent, high quality submission.

**Pull request merged**